source/conf/cmake/CMakeBuild.config.default

```diff
@@ -427,9 +427,6 @@ FLASH_MICROAPP_BASE=0x69000
 # The number of pages for microapps (each page is 0x1000 / 4096 bytes)
 FLASH_MICROAPP_PAGES=2
 
-# The start of where the microapp is allowed to put its memory (start of its .bss .data and then the heap)
-RAM_MICROAPP_BASE=0x2000B000
-
-# The amount of RAM it is allowed to use (stack grows down from RAM_MICROAPP_BASE + RAM_MICROAPP_AMOUNT)
-RAM_MICROAPP_AMOUNT=0x2000
+# The amount of RAM it is allowed to use (microapp is placed before IPC ram, and grows down).
```

Comment:
The comment "and grows down" is a bit confusing. The microapp is allocated a fixed amount `RAM_MICROAPP_AMOUNT` and will use parts at the beginning and at the end of this piece of ram.